webauth/index.js

```diff
@@ -86,7 +86,7 @@ export default class WebAuth {
             status: 0
           });
         }
-        return client.exchange({ code, verifier, redirectUri });
+        return client.exchange({ code, verifier, redirectUri }, options.nonce);
```

Comment:
Should the user of this library add the options.nonce?
e.g.
```
auth0
 .webAuth
 .authorize({scope: 'openid email', audience: 'https://{AUTH0_DOMAIN}/userinfo'}, nonce:<RANDOM_STRING>)
```
if so, how does [this](https://github.com/auth0/react-native-auth0/blob/eab2e1d6b3ad638fe772013cfdd63a280469594e/auth/__tests__/index.spec.js#L65-L68 ) test pass? 

If that's not the case and the nonce is not passed as an option by the user, how does it get a value?

Reply:
This test has nothing to do with the nonce.. it's only validating the constructor call.

I'm not really sure why I'm using `options.nonce` instead of something auto generated. I'll have to dig through the PR again to test it.

Reply:
Is there any update on that one? I'll then sign off from security.